### ISSUE TITLE
An error about auto pet mod

### ISSUE THREAD

**Issue by Fatalfuryy** [PsykoDev/Auto-Pet#4]
i'm trying to use this mod but always getting an error 
![auto pet error](https://user-images.githubusercontent.com/46627189/78446327-23ff9580-7682-11ea-8485-4a448ea6d840.jpg)

**Comment by PsykoDev** [comment]
have you copy past all .def in the def folder inside Proxy Tera\data\definitions ?

**Comment by Fatalfuryy** [comment]
yes i did. copied them but there are only 2 files in def folders and copied them to Proxy Tera\data\definitions

**Comment by PsykoDev** [comment]
opcode too ? https://github.com/PsykoDev/Full-Opcode-Tera-EU-NA

**Comment by Fatalfuryy** [comment]
so it's okay with auto fish ?

**Comment by PsykoDev** [comment]
try with that https://github.com/PsykoDev/Last-Def-Files

**Comment by andersonepk** [comment]
Worked fine after this 

try with that https://github.com/PsykoDev/Last-Def-Files


ty

**Comment by andersonepk** [comment]
My bad, acctualy is not working
i can load the mod in the game but the functions are not working.
He not using the skill with broch or feeding the pet.
He can sumon him only,

Can fix? 
ty
